app/controllers/v0/sessions_controller.rb

```diff
@@ -78,6 +110,7 @@ def async_create_evss_account(user)
     end
 
     def handle_completed_slo
+      saml_settings = saml_settings(name_identifier_value: @session&.uuid)
```

Comment:
How was SLO working before without the `name_identifier_value` part?

Reply:
Concerned because this looks like change to existing part of API

Reply:
hmm, actually this is a good catch, not sure it was. 

The old logic was all self contained in saml_settings, but it wasn't a clean interface in my opinion, so I refactored to make it more explicit on an action by action basis. FWIW, slo_callback gets called by id.me so pretty sure @session will be nil. and name_identifier will be nil as a result. I will remove even though it doesn't cause any issues to reduce confusion.

Reply:
Hmmmm... nevermind, handle_completed_slo does have session. I think it's helpful to provide this for debugging purposes. Basically it's a way for idme to link multiple user interactions together, beyond that I'm not sure it serves specific purpose. @omgitsbillryan could maybe provide more context. I'm leaving this in, since I think it's useful if available. I'm also providing it in multifactor and identityproof endpoints, with new/sessions?level=3 essentially deprecated, we have an opportunity to pass the session.uuid when we have it, and that is also why identity_proof (or the old /sessions/new?level=3) is now behind authorization requiring session token.

Reply:
@patrickvinograd the old code in saml_settings would set `name_identifier_value = session&.uuid` automatically if session was available. It's no different here.